video/README.md

```diff
@@ -2,9 +2,9 @@
 
 # Google Cloud Video Intelligence API Node.js Samples
```

Comment:
Ping me after approving and merging #384 to talk about auto-generating the video-intelligence samples README file.